docs/content_management/work_with_content_types.md

```diff
@@ -1,15 +1,22 @@
 ---
-description: Manage your website by modifying Content Types and fine-tuning access control by using the permission system.
+description: Build new Content Types and modify existing ones.
 ---
 
-# Organizing the site
+# Work with Content Types
```

Comment:
That's because the content is sort of a melting pot of ideas, too. This will change once we have an actual "Create Content Type" article. Then this article will change to "Manage Content Type settings"

Reply:
Also, splitting articles into individual sub-topics within tree branches is a great idea, we may do it in the future.